### PR TITLE
Add region name to the region list

### PR DESCRIPTION
**Pull request by dmakeroam**
- I added region name to the region list to show region name while selecting a region in the list. It makes a better experience.

Screenshot 2563-10-30 at 13 33 54

### CODE REVIEW

**Comment by dmakeroam** [comment]
@Issif @Shr3ps Please help to review

File: main.go [hunk 1]
```diff
@@ -143,25 +143,25 @@ func selectProfile(profiles []profile) profile {
...
+	"ap-southeast-2 | Sydney",
+	"ca-central-1 | Central",
+	"cn-north-1 | Beijing",
+	"cn-nortwest-1 | Ningxia",
+	"eu-central-1 | Frankfurt",
+	"eu-west-1 | Ireland",
+	"eu-west-2 | London",
+	"eu-west-3 | Paris",
+	"eu-north-1 | Stockholm",
+	"sa-east-1 | São Paulo",
```
Comment (Issif) [inline main.go]:
```suggestion
	"us-east-2      | Ohio",
	"us-east-1      | N. Virginia",
	"us-west-1      | N. California",
	"us-west-2      | Oregon",
	"ap-south-1     | Mumbai",
	"ap-northeast-3 | Osaka-Local",
	"ap-northeast-2 | Seoul",
	"ap-northeast-1 | Tokyo",
	"ap-southeast-1 | Singapore",
	"ap-southeast-2 | Sydney",
	"ca-central-1   | Central",
	"cn-north-1     | Beijing",
	"cn-nortwest-1  | Ningxia",
	"eu-central-1   | Frankfurt",
	"eu-west-1      | Ireland",
	"eu-west-2      | London",
	"eu-west-3      | Paris",
	"eu-north-1     | Stockholm",
	"sa-east-1      | São Paulo",
```

File: README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ You can select your profile/region by &larr;, &uarr;, &rarr; &darr; and filter b
 ## Build
 
 ```bash
-go build
+make all
```
Comment (Issif) [inline README.md:35]: It means you PR#3 is useless, right?

**Review by Issif — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dmakeroam) [reply README.md:35]: @Issif No, I would like PR #3 to be merged also to support binary release distribution.

**Comment by dmakeroam** [comment]
@Issif I updated source code as your suggestion already.

**Review by Issif — APPROVED** [review]

**Pull request merged**